flexget/plugins/daemon/irc.py

```diff
@@ -674,10 +674,13 @@ def entries_from_linepatterns(self, lines):
         entries = []
         for line in lines:
             # If it's listed in ignore lines, skip it
+            ignore = False
             for rx, expected in self.ignore_lines:
                 if rx.match(line) and expected:
                     log.debug('Ignoring message: matched ignore line')
-                    continue
+                    ignore = true
```

Comment:
Yep, just discovered this on my local instance! As mentioned I'm not a Python dev. Fix incoming